assets/info.plist

```diff
@@ -857,10 +857,22 @@ https://github.com/edgarjs/alfred-github-repos/issues</string>
 	</dict>
 	<key>variables</key>
 	<dict>
+		<key>CACHE_TTL_SEC_ORG</key>
+		<string>86400</string>
+		<key>CACHE_TTL_SEC_PR</key>
+		<string>300</string>
+		<key>CACHE_TTL_SEC_REPO</key>
+		<string>86400</string>
 		<key>GITHUB_ACCESS_TOKEN</key>
 		<string></string>
-		<key>GITHUB_HOST</key>
+		<key>GITHUB_API_HOST</key>
 		<string>api.github.com</string>
+		<key>GITHUB_HOST</key>
+		<string>www.github.com</string>
+		<key>GITHUB_ME_ACCOUNT</key>
+		<string>@me</string>
```

Comment:
some of the enterprise Github instance doesn't recognise `@me`, hence make this as an env variable.